sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -126,6 +126,8 @@ void gnrc_ipv6_demux(kernel_pid_t iface, gnrc_pktsnip_t *pkt, uint8_t nh)
                 gnrc_pktbuf_release(pkt);
                 return;
             }
+            /* gnrc_ipv6_ext_demux calls gnrc_ipv6_demux, so stop processing here */
+            return;
```

Comment:
This will keep external subscribers to any of those extension (e.g. raw sockets) from getting these packets, too. They are send in https://github.com/RIOT-OS/RIOT/pull/4654/files#diff-74976d45a42eaa063e3561b5fd36f615R141 ff.

Use `break` instead.